.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,57 @@
+name: Static Analysis
+on:
+  push:
+    branches:
+    - '**'
+    tags-ignore:
+    - v[0-9]+.[0-9]+.[0-9]+*
```

Comment:
I'm a bit afraid that this would allow unformatted code to just become a _tag_ without any messages.

Can we always run static analysis?